src/workerd/api/actor.h

```diff
@@ -179,6 +188,13 @@ class DurableObjectNamespace: public jsg::Object {
       JSG_METHOD(getExisting);
     }
     JSG_METHOD(jurisdiction);
+    if (flags.getDurableObjectDestroy()){
+      if (flags.getDurableObjectGetExisting()) {
+        JSG_METHOD(destroyExisting);
```

Comment:
Bikeshedding but IMO there's no need for the methods to have different names depending on whether `getExisting` is enabled. Note that you can define `destroy()` to take `CompatibilityFlags::Reader` as its last argument, and it'll be filled in automatically by the system, so you can then check the flag at runtime.